docs/en/stack/security/authorization/built-in-roles.asciidoc

```diff
@@ -111,6 +111,14 @@ user has access to only their own reports. Reporting users should also be
 assigned the `kibana_user` role and a role that grants them access to the data 
 that will be used to generate reports.
 
+[[built-in-roles-snapshot-user]] `snapshot_user`::
+Grants the necessary privileges to create snapshots of **all** the indices and
+to view their metadata. This role enables users to view the configuration of
+existing snapshot repositories and of the snapshot details. It does not enable
```

Comment:
Unless there's a "configuration of snapshot details", I think this should be changed to:

```suggestion
existing snapshot repositories and snapshot details. It does not enable
```